.circleci/config.yml

```diff
@@ -111,13 +132,54 @@ jobs:
           command: |
             echo "export IMAGE_VERSION_TAG=$(.circleci/get_version.sh)" >> workspace/env-vars
       - run: cat workspace/env-vars >> $BASH_ENV
-      - run: echo $IMAGE_VERSION_TAG
-      # - run: sleep 3m
+      - run: echo "IMAGE_VERSION_TAG=${IMAGE_VERSION_TAG}"
       - persist_to_workspace:
           root: workspace
           paths:
             - env-vars
 
+  should-publish:
+    docker:
+      - image: cimg/base:2022.08
+    working_directory: *work-dir
+    environment:
+      # £ delimited strings of regex for matches which should be published
+      PUBLISHABLE_TAGS: '^[0-9]+\.[0-9]+\.[0-9]+$'
+      PUBLISHABLE_BRANCHES: '^main$£^hotfix.*'
```

Comment:
@gjedlicska - not 100% sure about this here

Reply:
add alpha branch option